modules/distribution/src/assembly/bin.xml

```diff
@@ -259,6 +259,12 @@
             </includes>
         </fileSet>
 
+	<fileSet>
+            <directory>../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/dbscripts/bps</directory>
+            <outputDirectory>${pom.artifactId}-${pom.version}/dbscripts/bps</outputDirectory>
```

Comment:
Check if this can be renamed to dbscripts/business-process to refer the feature repo instead of the product repo. If not lets suggest BPS to change it.